src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -558,7 +563,7 @@ public static void BuildChain_WithApplicationPolicy_NoMatch()
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(SignatureSupport), nameof(SignatureSupport.SupportsRsaSha1Signatures))]
```

Comment:
Does this need to be conditionally run? Looks like it could just use that boolean as guards around the expected chain success?

Reply:
Yes, I'll do that instead.